package.json

```diff
@@ -95,8 +95,8 @@
   "peerDependencies": {
     "@emotion/react": "11.9.0",
     "@emotion/styled": "11.8.1",
-    "react": "17.0.2",
-    "react-dom": "17.0.2"
+    "react": "18.1.0",
+    "react-dom": "18.1.0"
```

Comment:
Shouldn't we specify `18.x` / `^18.0.0` instead of a specific version of the types?

Reply:
Well you're right, all of this package peerDeps are dumb :)